doctests

```diff
@@ -57,11 +59,22 @@ def command_to_converted_args(location, command):
     # Bit of a hack, but this will convert commands starting with 'wheatley' to start with './run-wheatley'
     # and make sure that [ID NUMBER] is replaced with a valid room ID.  This way, the tests will run on the
     # version contained in the current commit, rather than a release version.
-    edited_command = "./run-" + command.replace("[ID NUMBER]", ROOM_ID) \
+    edited_command = "python ./run-" + command.replace("[ID NUMBER]", ROOM_ID) \
                                        .replace("[METHOD TITLE]", '"' + EXAMPLE_METHOD + '"')
+    insert_arg_index = 2
+
+    # Check if running inside venv, if so we need to activate in each process
+    if sys.prefix != sys.base_prefix:
+        # Windows path `\` gets dropped, but Windows is happy to accept `/` instead
+        activate_script = sys.prefix.replace("\\", "/") + "/Scripts/activate"
```

Comment:
I tested this in a Windows sandbox with a clean python + venv install. 

My WSL ubuntu is refusing to create a venv to test this also works there on linux, but it looks about right

Reply:
Yep I'm not using venv so this looks good.